movie-details-app/pages/index.js

```diff
@@ -0,0 +1,93 @@
+import { useState } from "react";
+import Image from "next/image";
+import Link from "next/link";
+import axios from "axios";
+
+export default function Home() {
+  const [movieName, setMovieName] = useState(null);
+  const [movieInfo, setMovieInfo] = useState(null);
+
+  /**
+   *
+   *
+   * Fetch Movie Info from the IMDB
+   */
+  const fetchMovieInfo = async () => {
+    try {
+      const res = await axios.get(`http://localhost:3000/api/movie`, {
```

Comment:
This won't work when you deploy the application.

Reply:
@ghoshnirmalya I have deployed the application from a separate repo. I didn't change it this time because I thought when someone clones this repo, and run it locally, it will fail if I have the deployed link here because of the cross-origin error.

Let me know your thoughts about it.

Reply:
In that case, the URL can be an environment variable. In **.env.local**, it will be localhost:3000. But, on production, it will be different.